src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx

```diff
@@ -21,7 +21,13 @@ const ProjectsTable: React.FC = () => {
   const { isLoading, data, error } = useAllProjects();
   const [pageSize, setPageSize] = useState(30);
 
-  const baseColDef: any = {
+  interface BaseColDef {
```

Comment:
Great work here! Can we just have this interface located above the render name? So just put this interface above line 19.